src/index.html

```diff
@@ -11,7 +11,173 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+    >
+      <fieldset>
+        <legend>Personal information</legend>
+
+        <label class="form-field">
+          Surname:
+          <input
+           type="text"
+           name="surname"
+           autocomplete="off"
+           required
+          >
+        </label>
+
+        <label class="form-field">
+          Name:
+          <input
+           type="text"
+           name="name"
+           autocomplete="off"
+           required
+          >
+        </label>
+
+        <label class="form-field">
+          How old are You?
+          <input
+           type="number"
+           name="age"
+           min="1"
+           max="100"
+           value="12"
+           required
+          >
+        </label>
+
+        <label class="form-field">
+          Full date of birth:
+          <input
+           type="date"
+           name="dateOfBirth"
+          >
+        </label>
+
+        <label>
+          I accept the term of the agreement
+          <input
+           type="checkbox"
+           name="terms"
+           required
+          >
+        </label>
+      </fieldset>
+
+      <fieldset>
+        <legend>Registration</legend>
+
+        <label class="form-field">
+          E-mail:
+          <input
+           type="email"
+           name="email"
+           data-qa="field-placeholder"
+           placeholder="email@example.com"
+          >
+        </label>
+
+        <label>
+          Password:
+          <input
+           type="password"
+           name="password"
+           minlength="7"
+           maxlength="14"
+          >
+        </label>
+      </fieldset>
+
+      <fieldset>
+        <legend>An interesting fact about you!</legend>
+
+        <label class="form-field">
+          Do you love cats?
+
+          <input
+           type="radio"
+           name="loveCats"
+          >
+          Yes
+
+          <input
+           type="radio"
+           name="dontLoveCats"
+          >
+          No
+        </label>
```

Comment:
Each input should have a label here. And `Do you love cats?` use outside a label

Reply:
is not fixed